lib/tasks/cicognara.rake

```diff
@@ -0,0 +1,12 @@
+require 'cicognara/tei_indexer'
+
+namespace :tei do
```

Comment:
Please rename the file to `tei.rake` to make it easier to figure out where the `tei:index` command is defined.